### ISSUE TITLE
When clicking on Edit Comment, there is no way to cancel the edit; it always stays open.

### ISSUE THREAD

**Issue by omarNaifer12** [docmost/docmost#579]
![docCancel](https://github.com/user-attachments/assets/d04d3cf6-0c7c-4d9a-971d-5a00e47d75ca)

**Comment by Philipinho** [comment]
Want to work on this?

**Comment by omarNaifer12** [comment]
Yes, should I just add a red cancel button with the same type as the save button?

**Comment by Philipinho** [comment]
Let's use the default variant. See https://mantine.dev/core/button/.

**Comment by omarNaifer12** [comment]
I sent the PR. Could you review it along with the other PR for the emoji list to see if anything needs to be adjusted?

**Comment by Philipinho** [comment]
I am working a a branch. I will eventually get to the PRs.
Thank you for your patience.

**Comment by Philipinho** [comment]
Fixed here https://github.com/docmost/docmost/pull/580/.
